velox/dwio/parquet/writer/arrow/Properties.h

```diff
@@ -323,10 +323,10 @@ class PARQUET_EXPORT WriterProperties {
           write_batch_size_(DEFAULT_WRITE_BATCH_SIZE),
           max_row_group_length_(DEFAULT_MAX_ROW_GROUP_LENGTH),
           pagesize_(kDefaultDataPageSize),
-          version_(ParquetVersion::PARQUET_2_6),
+          version_(ParquetVersion::PARQUET_1_0),
           data_page_version_(ParquetDataPageVersion::V1),
           created_by_(
-              std::string("parquet-mr version 2.6.0 (build ") +
+              std::string("parquet-mr version 1.13.1 (build ") +
```

Comment:
This is incorrect. This should be `parquet-cpp-velox version 0.0.0`. We can add the version number once when we have Velox versioning.
We should also allow users to be able to specify a string for this. See https://github.com/apache/arrow/blob/c56ec12a2f38ed6fce2d87ea65345b6edf4234f5/cpp/src/parquet/properties.h#L751